src/main/resources/swagger/rawls.yaml

```diff
@@ -3247,6 +3247,35 @@ paths:
             - email
             - profile
             - https://www.googleapis.com/auth/devstorage.full_control
+  '/api/workspaces/{workspaceNamespace}/{workspaceName}/getBucketUsage':
```

Comment:
It just occurred to me that "getBucketUsage" is a horrible name because "get" is redundant with the HTTP method. I'll change this to "bucketUsage" unless someone can suggest a better name. Since this is an exported API, others who have been around longer probably have stronger feelings about this.